guides/serial.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: guide
+banner: images/coolterm-banner.png
```

Comment:
can you make a banner that doesn't have white at the bottom? (where does the image end, and where does the guide start?!)

![image](https://cloud.githubusercontent.com/assets/578029/6419361/b4dd8d42-bebc-11e4-966c-eb1647de6e3d.png)